src/pages/settings/Profile/PersonalDetails/LegalNamePage.js

```diff
@@ -68,34 +68,20 @@ class LegalNamePage extends Component {
         const errors = {};
 
         // Check for invalid characters in legal first and last name
-        const [invalidLegalFirstNameCharacter, invalidLegalLastNameCharacter] = ValidationUtils.findInvalidSymbols(
-            [values.legalFirstName, values.legalLastName],
-        );
-        const [hasLegalFirstNameLengthError, hasLegalLastNameLengthError] = ValidationUtils.doesFailCharacterLimitAfterTrim(
-            CONST.LEGAL_NAMES_CHARACTER_LIMIT,
+        const [doesLegalFirstNameHaveInvalidCharacters, doesLegalLastNameHaveInvalidCharacters] = ValidationUtils.doesContainCommaOrSemicolon(
             [values.legalFirstName, values.legalLastName],
         );
 
-        if (!_.isEmpty(invalidLegalFirstNameCharacter)) {
-            errors.legalFirstName = Localize.translateLocal(
-                'privatePersonalDetails.error.hasInvalidCharacter',
```

Comment:
`privatePersonalDetails.error.hasInvalidCharacter` is not used anymore.

Reply:
Right you are.